<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign_Up</class>
 <widget class="QDialog" name="Sign_Up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="confirm_label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>470</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>1171</width>
     <height>631</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="validPass">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>280</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>310</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter a password:*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Sign_Up_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>390</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newPass">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>300</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="validUser">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>180</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>210</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter username:*</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="confirmation">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>440</y>
       <width>211</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm if you are a human</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Sign_In">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Go back to Sign In</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newUser">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>200</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>confirm_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
